boards/arduino/portenta_h7/arduino_portenta_h7_stm32h747xx_m7.dts

```diff
@@ -15,39 +15,68 @@
 
 	/* HW resources are split between CM7 and CM4 */
 	chosen {
-		zephyr,console = &usart1;
-		zephyr,shell-uart = &usart1;
+		zephyr,console = &cdc_acm_uart0;
+		zephyr,shell-uart = &cdc_acm_uart0;
+		zephyr,cdc-acm-uart0 = &cdc_acm_uart0;
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
-		zephyr,code-partition = &code_partition;
+		zephyr,code-partition = &slot0_partition;
 	};
 
 	oscen: oscen {
 		compatible = "regulator-fixed";
 		regulator-name = "oscen";
 		enable-gpios = <&gpioh 1 GPIO_ACTIVE_HIGH>;
 		regulator-boot-on;
+		status = "okay";
+	};
+
+	ethernet_phy_en: ethernet_phy_en {
+		compatible = "regulator-fixed";
+		regulator-name = "ethernet-phy-reset-release";
+		enable-gpios = <&gpioj 15 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+		status = "okay";
+	};
+
+	sdram1: sdram@c0000000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		device_type = "memory";
+		reg = <0xc0000000 DT_SIZE_M(8)>;
+		zephyr,memory-region = "SDRAM1";
+		zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM) )>;
 	};
 };
 
+&clk_lse {
+	clock-frequency = <32768>;
+	status = "okay";
```

Comment:
This is unfortunately not always the case. Early Portenta H7 boards had production issues that resulted in the 32768 Hz oscillator either _missing_ or _badly working_. The Mbed core [works around this issue at runtime](https://github.com/arduino/ArduinoCore-mbed/blob/dd5abb6171063b2777620b9339510c63b915ce72/variants/PORTENTA_H7_M7/variant.cpp#L316-L337) by basically making sure the board is new enough or that the LSE clock is in step with the HSE - otherwise it defaults to LSI. 

Not sure how should we handle this in Zephyr. Should we provide 2 board revisions, with this line as the difference, and let the user decide?

Reply:
In this case I believe the build system has support for [board revisions](https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#porting-board-revisions). Would it make sense to have it defined like that? If you have a documentation the lists the hardware changes between the various revision I can make the necessary changes.

Reply:
I think so. As the core does, we can define a 1.0 default (without LSE) and a 4.10 that enables it if the user manually chooses. We should mention this in the board's `doc/index.rst` file as well!